config/jest/jest.config.js

```diff
@@ -12,25 +12,26 @@ const fs = require('fs');
 const path = require('path');
 const {optionParser: app} = require('@enact/dev-utils');
 
+const rbConst = name =>
+	'ILIB_' +
+	path
+		.basename(name)
+		.replace(/[-_\s]/g, '_')
+		.toUpperCase() +
+	'_PATH';
+
 const iLibDirs = ['node_modules/@enact/i18n/ilib', 'node_modules/ilib', 'ilib'];
 const globals = {
 	__DEV__: true,
 	ILIB_BASE_PATH: iLibDirs.find(f => fs.existsSync(path.join(app.context, f))) || iLibDirs[1],
 	ILIB_RESOURCES_PATH: 'resources',
-	ILIB_CACHE_ID: new Date().getTime() + ''
+	ILIB_CACHE_ID: new Date().getTime() + '',
+	[rbConst(app.name.split('/').pop())]: 'resources'
```

Comment:
I suspect `split` and `pop` are redundant here with the `basename` call in `rbConst`.